db/seeds.rb

```diff
@@ -7,7 +7,9 @@
   original_text_parsed   = doc.css("#table.table tbody tr td")[i].text.strip
   translated_text_parsed = doc.css("#table.table tbody tr td")[i + 2].text
   i += 5
-  Card.create(translated_text: translated_text_parsed, 
-              original_text:   original_text_parsed, 
-              review_date:     date)
+  card = Card.create(translated_text: translated_text_parsed, 
+                  original_text:   original_text_parsed, 
```

Comment:
Align the elements of a hash literal if they span more than one line.<br>Unnecessary spacing detected.<br>Trailing whitespace detected.